pplns-with-job-declaration.tex

```diff
@@ -21,174 +21,193 @@
+
+Given two miners with the same hashrate that submit the same amount of work to the pool, but each has different block templates via Job Declaration. The first miner optimizes transaction selection for maximum fees, and the second not so much. With traditional PPLNS, these two miners are paid equally since only hashrate (work submitted via shares) is considered. This is unfair as the miner with optimized transaction fees is working toward higher block reward for the pool, and should be rewarded commensurately.
 
-In this section we aim to solve this issue. The idea of our proposal is to split the payout of each share, and redistribute the coinbase reward and the block fee separately. Since the coinbase reward is independend on the other transactions in the block, it is also independent from Job Declaration, and therefore will be redistributed in base of the difficulty score $score_d(s_i)$ of the $i$-th share $s_i$. So, the coinbase reward will be redistributed in the same fashion as standard PPLNS. \newline
-For the block difficulty, we must introduce a method to weight shares based on the fees. We must neither lose the dependency on the difficulty, because in the future miners are supposed to be paid mainly with fees. So, the block fees are supposed to be redistributed amongs miners based on a score $score_d(s_i)$ that depends both difficulty and the fees of every share.
+To solve this inequity, we split the payout of each share, and score the block subsidy and the block fee differently. Since the block reward is determined independent of other transactions in the block, we also consider it independently from Job Declaration. We distribute the block reward based on traditional PPLNS share scoring: $score_d(s_i)$ of the $i$-th share $s_i$.\newline
+For Job Declaration, we must introduce a new method to calculate $score_d(s_i)$ based on the fees for a submitted share, while retaining the score based on the work of submitted shares. This is desireable as the block subsidy trends to zero and fees become the predominate value of a block reward. 
 
-In the final subsection it will be given the precise definition for the payout of $s_i$.
+In the final subsection precise definition for the payout of $s_i$ is given.
 
 \subsection{Difficulty-based score}
 If the window contains $N$ shares, then for the $i$-th share $s_i$ we define its score
 \[score_d(s_i) = \frac{d_i}{\sum_{j=1}^Nd_j},\]
-where $d_j$ is the difficulty of the share $s_j$ (for example, \cite{ocean} suggest to adopt $N$ such that the sum at the denominator eight times the bitcoin difficulty).\newline
+where $d_j$ is the difficulty of the share $s_j$ (for example, Ocean TIDES\cite{ocean} suggest $N$ be the sum at the denominator eight times the bitcoin difficulty).\newline
 \begin{remark}
-	Note that
-	\[\sum_{j=1}^N score_d(s_i) = 1\] and that this score depends only on difficulty.
+	Note that \[\sum_{j=1}^N score_d(s_i) = 1\] depends only on difficulty.
```

Comment:
I found I typo reading this change. The summation index should be $i$ and not $j$. Thank you!